javagen/src/main/java/com/azure/autorest/template/example/ModelExampleWriter.java

```diff
@@ -309,9 +313,15 @@ public String accept(ExampleNode node) {
                     }
                 }
                 return builder.toString();
+            } else if (node instanceof BinaryDataNode) {
+                this.imports.add(com.azure.core.util.BinaryData.class.getName());
+                return binaryDataNodeExpression((BinaryDataNode) node);
             }
             return null;
         }
     }
 
+    private static String binaryDataNodeExpression(BinaryDataNode binaryDataNode) {
+        return String.format("BinaryData.fromBytes(\"%s\".getBytes())", binaryDataNode.getExampleValue());
```

Comment:
You probably need some escaping. E.g. what if example value be `"example"` with quote?

Reply:
Thanks, added escape and test case.